pkg/handlers/health_check.go

```diff
@@ -20,13 +21,22 @@ const (
 )
 
 type healthCheck struct {
-	Status status `json:"status"`
+	Status    status `json:"status"`
+	Datetime  string `json:"datetime"`
+	Version   string `json:"version"`
+	Timestamp string `json:"timestamp"`
 }
 
 // Handle handles a web request and returns a healthcheck JSON payload
 func (h HealthCheckHandler) Handle() http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		js, err := json.Marshal(healthCheck{Status: statusPass})
+		statusReport := healthCheck{
+			Status:    statusPass,
+			Version:   os.Getenv("APPLICATION_VERSION"),
+			Datetime:  os.Getenv("APPLICATION_DATETIME"),
```

Comment:
What's the reason for having datetime and timestamp?

Reply:
The first is for human consumption. A unix timestamp makes it easy to calculate how long a build has been deployed, automate scanning for old builds etc.

We will have to redeploy containers on some kind of schedule, and so I thought it would be harmless to include this now to aid with verification.